docs/tools-cli.md

```diff
@@ -0,0 +1,120 @@
+---
+title: CLI Helpers
+---
+
+If you end up working with large data sets such as the latest dump from Singular, we recommend using the CLI tools to generate this json data for you, in case you experience a large CPU data spike when testing out your frontend. You can read the original CLI documentation below or in the [rmrk-tools github repo](https://github.com/rmrk-team/rmrk-tools).
+
+### How to install CLI helpers
+
+You can use this package as a CLI tool: `yarn add rmrk-tools@latest --dev`
+
+Now you can use any of the helper commands in your bash or npm scripts by prepending `rmrk-tools-` to the helper name:
+
+```
+"scripts": {
+    "fetch": "rmrk-tools-fetch",
+    "consolidate": "rmrk-tools-consolidate",
+    "seed": "rmrk-tools-seed",
+}
+```
+
+Or in bash:
+
+```
+#! /usr/bin/env node
+import shell from "shelljs";
+
+shell.exec(
+  'rmrk-tools-fetch --ws wss://node.rmrk.app --prefixes=0x726d726b,0x524d524b --append=dumps/latest.json',
+);
+```
+
+### Additional notes
+
+- If you get a build error when you try using the cli helper tools, you need to make sure you install all of your dependencies (for me, it required that I install the dependencies `@polkadot/api` and `arg`).
```

Comment:
If we need these and they are not included, this is something to be flagged on the tools repo and fixed.